test/jest/__tests__/api/fuel.spec.ts

```diff
@@ -0,0 +1,340 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import { AnyTransaction } from '@greymass/eosio';
+import { describe, expect, it, jest, beforeEach } from '@jest/globals';
+import { installQuasarPlugin } from '@quasar/quasar-app-extension-testing-unit-jest';
+import { FuelUserWrapper } from 'src/api/fuel';
+import {
+  SignTransactionConfig,
+  SignTransactionResponse,
+  User
+} from 'universal-authenticator-library';
+
+installQuasarPlugin();
+
+// mocking localStorage
+// https://robertmarshall.dev/blog/how-to-mock-local-storage-in-jest-tests/
+const localStorageMock = (function () {
+  let store: { [key: string]: unknown } = {};
+
+  return {
+    getItem(key: string) {
+      return store[key];
+    },
+
+    setItem(key: string, value: unknown) {
+      store[key] = value;
+    },
+
+    clear() {
+      store = {};
+    },
+
+    removeItem(key: string) {
+      delete store[key];
+    },
+
+    getAll() {
+      return store;
+    }
+  };
+})();
+Object.defineProperty(window, 'localStorage', { value: localStorageMock });
+
+// TODO: this do not work. How do you mock a class to work with: if (user instanceof AnchorUser) {...}
+// mocking AnchorUser
```

Comment:
remove if it doesn't work, keep a local copy, we can re-visit.

Reply:
I did remove the complete stub [19c0643](https://github.com/telosnetwork/open-block-explorer/pull/477/commits/19c0643094757cebd2c4df92008a58744db7d3f4), but it was actually working to mock the class so I had the restore it [c9ca416](https://github.com/telosnetwork/open-block-explorer/pull/477/commits/c9ca4163bed9d2ca9c4f214f28f794ef10a154fe).

It just does not work when trying to mock the `instaceof` operator, but the normal use of mocking the class does work.